ferrocene/doc/internal-procedures/src/figures/ci-failure-2.png.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: The Ferrocene Developers
+
```

Comment:
nit: needless space

same applies to other license files in this pr

Reply:
I used `reuse` so I assumed it was the expected format.

Reply:
it's not wrong... it's just not consistent

Reply:
Ok done... I wonder if we can fix the tool to autogenerate with the right format...